src/PlayingBoard.re

```diff
@@ -0,0 +1,155 @@
+[%bs.raw {|require('./playingBoard.css')|}];
+open SharedTypes;
+
+type playingBoard = array(array(env));
+
+let addRock = (num: int, field: env) =>
+  switch (num) {
```

Comment:
This can be extracted to standalone function, you are using it on several places. 
```javascript
switch (num) {
  | 1 => SomeType
  | _ => field
}